File: tasks/verticle/templates/verticle.java
package {{ groupId }};

import io.vertx.core.AbstractVerticle;

public class {{ verticleName }} extends AbstractVerticle {

  @Override
  public void start() {
    // your code goes here...
    vertx.createHttpServer().requestHandler(req -> {
      req.response()
        .putHeader("content-type", "text/plain")
        .end("Hello from Vert.x!");
    }).listen(8080);
  }
}

File: tasks/verticle/templates/verticle.kt
package {{ groupId }}

import io.vertx.core.AbstractVerticle

class {{ verticleName }} : AbstractVerticle() {

  override fun start() {
    // your code goes here...
    vertx.createHttpServer().requestHandler({ req ->
      req.response()
        .putHeader("content-type", "text/plain")
        .end("Hello from Vert.x!")
    }).listen(8080)
  }
}
